### DISCUSSION TITLE
All the GSSoC'21 Contributors

### DISCUSSION THREAD

**Discussion by vatsalkesarwani12** [Algo-Phantoms/Algo-Phantoms-Frontend#11] [Q&A] [answered]
When to start contributing?

**Comment by vatsalkesarwani12** [comment, marked as answer]
First of all, Welcome to Algo Phantoms👋

All the contribution to Algo-Phantoms-Frontend are welcome but contributing to making a leaderboard in GSSoC'21 you must raise PR's from `1st March` to `31st May`.

Thank you :)
Happy Coding !!

**Reply by maanpatel2492** [reply]
Hi,I have solved some of the issue but it is already assigned to someone.can I still make a pull request?

**Comment by aarushi31** [comment]
Hi I'm a GSSoC'21 participant. I would like to contribute to this project. Is there a discord server for this so that I can join that?

**Reply by vatsalkesarwani12** [reply]
You will provided with that link soon by official gssoc

**Comment by Miru-s** [comment]
Hey I'm a GSSoC'21 participant, I would love to contribute to this project. Can you please provide me some issue and guide me further.

**Reply by harikesh409** [reply]
You are free to choose any of the open issues or if you have a new idea you can raise a new one.

**Comment by tier3guy** [comment]
What is the meaning of assigning an issue to someone ? Can others do contribute to those issue ?

**Reply by harikesh409** [reply]
Each issue will be assigned to a person and that person will work on that issue. There might be some cases where the issue will be assigned to work to multiple people to work as a team.
